tree-sitter-stack-graphs/src/lib.rs

```diff
@@ -635,22 +636,23 @@ impl<'a> Builder<'a> {
         let tree = parse_errors.into_tree();
 
         let mut globals = Variables::nested(globals);
+
         if globals.get(&ROOT_NODE_VAR.into()).is_none() {
             let root_node = self.inject_node(NodeID::root());
             globals
                 .add(ROOT_NODE_VAR.into(), root_node.into())
                 .expect("Failed to set ROOT_NODE");
         }
+
         let jump_to_scope_node = self.inject_node(NodeID::jump_to());
         globals
             .add(JUMP_TO_SCOPE_NODE_VAR.into(), jump_to_scope_node.into())
             .expect("Failed to set JUMP_TO_SCOPE_NODE");
-        if globals.get(&FILE_PATH_VAR.into()).is_none() {
-            let file_name = self.stack_graph[self.file].to_string();
-            globals
-                .add(FILE_PATH_VAR.into(), file_name.into())
-                .expect("Failed to set FILE_PATH");
-        }
+
+        // FILE_PATH is mandatory
+        globals
+            .get(&FILE_PATH_VAR.into())
+            .expect("FILE_PATH not set");
```

Comment:
I don't know if this is the right approach. While `FILE_PATH` is required in all TSG's we actually use, there's nothing inherently here that requires it.

I'm thinking we might just keep the old code here. It will act as a fallback and we don't unnecessarily introduce a breaking change here.